terraform/orphaned_backups_view.tf

```diff
@@ -0,0 +1,45 @@
+resource "google_bigquery_dataset" "orphaned_backups_dataset" {
+  project = "${var.bbq_project}"
+  dataset_id = "${var.orhpaned_backups_views_dataset}"
+  location = "${var.SLI_views_location}"
+
+  labels {"bbq_metadata"=""}
+
+  access {
+    role   = "WRITER"
+    special_group = "projectWriters"
+  }
+  access {
+    role   = "OWNER"
+    special_group = "projectOwners"
+  }
+  access {
+    role   = "WRITER"
+    special_group = "projectReaders"
+  }
+}
+
+resource "google_bigquery_table" "orphaned_backups" {
+  project = "${var.bbq_project}"
+  dataset_id = "${var.orhpaned_backups_views_dataset}"
+  table_id = "orphaned_backups_view"
+
+  view {
+    query = <<EOF
+          SELECT tableId, datasetId, projectId
+          FROM
+            [${var.gcp_census_project}:bigquery_view.table_metadata_deduplicated_aggregated]
```

Comment:
it's dangerous. 
let's consider this situation: 
1. datastore_export is made ( it's a cron, eg.every 6hours) and has data from 0:00
2. backup for new table is made (eg. 0.15)
3. census is taking snapshot which contains newly created table (eg. 0.30) (census has data about newly created table in table_metadata_deduplicated_aggregate view)

if this query will be executed, it returns newly created table as an orphaned backup (and we will delete it after all). 
Its bug - it will produce situation when we are thinking that we have backup (it is in datastore - which will be exported later), but we don't have the backup table

Reply:
done